src/app/core/models/sparque/sparque-suggest.interface.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Interface for Sparque Suggestions API response object
+ */
+export interface SparqueSuggestions {
+  products?: SparqueProduct[];
+  categories?: SparqueCategory[];
+  brands?: SparqueBrand[];
+  keywordSuggestions?: SparqueKeywordSuggestions[];
```

Comment:
for solr map searchterm to keywordSuggestions . rename keywordSuggestions to searchTerm